extra/man/alacritty-bindings.5.scd

```diff
@@ -0,0 +1,478 @@
+ALACRITTY-BINDINGS(5)
+
+# NAME
+
+Alacritty Bindings - Default configuration file bindings
+
+# SYNOPSIS
+
+This page documents all key and mouse bindings for the default Alacritty
+configuration. See *alacritty*(5) for full configuration format documentation.
+
+# Mouse Bindings
+
+[[ *mouse*
+:[ *mods*
+:[ *action*
+|  _"Right"_
+:[
+:  _"ExpandSelection"_
+|  _"Right"_
+:  _"Control"_
+:  _"ExpandSelection"_
+|  _"Middle"_
+:  _"~Vi"_
+:  _"PasteSelection"_
+
+# Key Bindings
+
+[[ *key*
+:[ *mods*
+:[ *mode*
+:[ *action* / *chars*
+|  _"Paste"_
+:[
+:[
+:  _"Paste"_
+|  _"Copy"_
+:[
+:[
+:  _"Copy"_
+|  _"L"_
+:  _"Control"_
+:[
+:  _"ClearLogNotice"_
+|  _"L"_
+:  _"Control"_
+:  _"~Vi|~Search"_
+:  *chars*: _"\x0c"_
```

Comment:
I'd assume you've ensured that `\x` bindings do work? I remember they were not a part of initial spec.

Reply:
They do not work… yet.